widget/label.go

```diff
@@ -92,15 +105,17 @@ func (l *Label) Refresh() {
 	l.BaseWidget.Refresh()
 }
 
-// Resize sets a new size for the label.
-// This should only be called if it is not in a container with a layout manager.
+// SelectedText returns the text currently selected in this Label.
+// If the label is not Selectable it will return an empty string.
+// If there is no selection it will return the empty string.
 //
-// Implements: fyne.Widget
-func (l *Label) Resize(s fyne.Size) {
-	l.BaseWidget.Resize(s)
-	if l.provider != nil {
-		l.provider.Resize(s)
+// Since: 2.6
+func (l *Label) SelectedText() string {
+	if !l.Selectable {
+		return ""
 	}
+
+	return l.selection.SelectedText()
```

Comment:
I think we need a nil check around l.selection in case SelectedText is called before the renderer is created